Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -426,26 +440,42 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     open override func startTunnel(options: [String: NSObject]?, completionHandler: @escaping (Error?) -> Void) {
         connectionStatus = .connecting
-        tunnelHealth.isHavingConnectivityIssues = false
-        controllerErrorStore.lastErrorMessage = nil
```

Comment:
When starting NetP on-demand we no longer clear these, since we consider there to be an ongoing issue.

The new logic is implemented in `resetIssueStateOnTunnelStart(_:)` which is called a few lines below.